ample/util/argparse_util.py

```diff
@@ -1,43 +1,47 @@
-"""
-Below is code for creating a boolean argument parser with a default, and a range of choices
+__author__ = "Jens Thomas & Felix Simkovic"
+__date__ = "10 June 2019"
+__version__ = "1.0"
+
+import argparse
+import os
+from ample.modelling.multimer_definitions import MULTIMER_MODES
+
 
 class BoolAction(argparse.Action):
```

Comment:
I think this code looks fine, however it might also be worth adding a deprecation warning in here. That way we can change it to a simpler 'store action True' method in due course but users won't be caught out.